app/service/file_svc.py

```diff
@@ -70,6 +70,18 @@ async def create_exfil_sub_directory(self, dir_name):
             os.makedirs(path)
         return path
 
+    async def create_exfil_operation_directory(self, dir_name, agent_name):
+        op_list = self.data_svc.ram['operations']
+        op_list_filtered = [x for x in op_list if x.state not in x.get_finished_states()]
+        timestamp_chars = str.maketrans({'T': '_', ':': ''})
+        special_chars = {ord(c): '_' for c in ':<>"/\|?*'}
```

Comment:
This is triggering W605 linting error - I think if the `\` is designed to be literal, the string should be raw (e.g. `r':<>"/\|?*'`)

Reply:
Corrected linting issue